judo-ui-react/src/main/java/hu/blackbelt/judo/ui/generator/react/UiWidgetHelper.java

```diff
@@ -437,6 +452,10 @@ public static Integer calculateLinkAutocompleteRows(Link link) {
         return defaultValue != null ? defaultValue : 10;
     }
 
+    public static Integer calculateTableAutocompleteRows(Table table) {
+        return 10;
+    }
```

Comment:
Shouldn't the value for this come from the model?

Reply:
No option yet in esm.

Reply:
Then I suggest creating a ticket to be able to keep it consistent with other autocomplete row counts.